data/50unattended-upgrades.Ubuntu

```diff
@@ -11,6 +11,8 @@ Unattended-Upgrade::Allowed-Origins {
 	// available, the policy for updates is such that unattended-upgrades
 	// should also install from here by default.
 	"${distro_id}ESM:${distro_codename}";
```

Comment:
Please do drop the old (and now unmatching) "${distro_id}ESM:${distro_codename}"